content/en/docs/external-add-ons/elyra/introduction.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Introduction"
+description = "A brief introduction to Elyra"
+weight = 10
++++
+
```

Comment:
We need Elyra folks to review this change.
Please take a look.
/assign @issheng @hsinhoyeh @akfmdl @akchinSTC

Reply:
cc @lresende

Reply:
left comment above